README.mkdn

```diff
@@ -152,9 +152,9 @@ application described by the TOML file.
 - `cargo xtask dist app/demo-stm32f4-discovery/app.toml` - stm32f4-discovery
 - `cargo xtask dist app/demo-stm32f4-discovery/app-f3.toml` - stm32f3-discovery
 - `cargo xtask dist app/lpc55xpresso/app.toml` - lpcxpresso55s69
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g031.toml` - stm32g031
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g070.toml` - stm32g070
-- `cargo xtask dist app/demo-stm32g0-nucleo/app-g0b1.toml` - stm32g0b1
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g031.toml` - stm32g031-nucleo
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g070.toml` - stm32g070-nucleo
+- `cargo xtask dist app/demo-stm32g0-nucleo/app-g0b1.toml` - stm32g0b1-nucleo
```

Comment:
Nit: `app/demo-stm32g0-nucleo/app-g0b1.toml` was renamed to `app/demo-stm32g0-nucleo/app-g0b1.toml.noworky` back in c12d361ebf